src/ide.js

```diff
@@ -747,13 +745,17 @@ D.IDE = function IDE(opts = {}) {
       // I.sb_trap.innerText = `⎕TRAP: ${x.TRAP}`; // TRAP doesn't display a value
       I.sb_dq.innerText = `⎕DQ: ${x.DQ}`;
       I.sb_threads.innerText = `&: ${x.NumThreads}`;
+      I.sb_cc.innerText = `CC: ${x.CompactCount}`;
+      I.sb_gc.innerText = `GC: ${x.GarbageCount}`;
       // Eventually we would like to read the default values from the interpreter.
       I.sb_ml.classList.toggle('active', x.ML !== 1);
       I.sb_io.classList.toggle('active', x.IO !== 1);
       I.sb_sis.classList.toggle('active', x.SI > 0);
       I.sb_trap.classList.toggle('active', x.TRAP !== 0);
       I.sb_dq.classList.toggle('active', x.DQ !== 0);
       I.sb_threads.classList.toggle('active', x.NumThreads > 1);
+      I.sb_cc.classList.toggle('active', x.CompactCount !== 1);
+      I.sb_gc.classList.toggle('active', x.GarbageCount !== 0);
```

Comment:
These 2 will only ever increase, so either don't make them active or (if we want to indicate when an increase happens) activate them on increase and then let them fade back after some time.